dashboard/src/main/java/org/apache/uniffle/dashboard/web/resource/Response.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.uniffle.dashboard.web.resource;
+
+public class Response<T> {
```

Comment:
Could we move it to the common module? And the same class in the coordinator module.

Reply:
> Could we move it to the common module? And the same class in the coordinator module.

This Response is in the Coordinator module. The Coordinator module is introduced into the dashboard here, which feels a bit awkward. Just like you said, it needs to be migrated to the common module, but I don't think it should be done in this PR.

Reply:
> > Could we move it to the common module? And the same class in the coordinator module.
> 
> This Response is in the Coordinator module. The Coordinator module is introduced into the dashboard here, which feels a bit awkward. Just like you said, it needs to be migrated to the common module, but I don't think it should be done in this PR.

I changed it when I introduced the previous code check.

Reply:
> > Could we move it to the common module? And the same class in the coordinator module.
> 
> This Response is in the Coordinator module. The Coordinator module is introduced into the dashboard here, which feels a bit awkward. Just like you said, it needs to be migrated to the common module, but I don't think it should be done in this PR.

I'm introducing front-end code inspection to change this.